task31.js

```diff
@@ -0,0 +1,11 @@
+import * as process from 'process';
```

Comment:
You can change origin files with ES6 feature without creating new files

Reply:
done